README.md

```diff
@@ -1 +1,55 @@
-# irods_client_http_python
+# iRODS HTTP API Python Wrapper
+
+This is a Python wrapper for the [iRODS HTTP API](https://github.com/irods/irods_client_http_api). 
+
+Documentation for the endpoint operations can be found [here](https://github.com/irods/irods_client_http_api/blob/main/API.md).
+
+## Setup
+**NOTICE:** This project is not yet available through pip. To use, clone the repository into the desired location.
+```
+git clone https://github.com/irods/irods_client_http_python.git
+```
+## Usage
+To use the wrapper, follow the steps listed below.
+
+Import the package.
+```
+from irods_client import IrodsClient
+```
+
+Create an instance of the wrapper with the base url of the iRODS server to be accessed. \<host>, \<port>, and \<version> are placeholders, and need to be replaced by appropriate values.
+```
+api = IrodsClient('http://<host>:<port>/irods-http-api/<version>')
+```
```

Comment:
Let's combine all of these code statements into one code snippet.

You can convert the descriptive lines to python comments within the code snippet.

For example _(some code removed for brevity)_:
```py
from irods_client import IrodsClient

# Create an instance of the wrapper ...
api = IrodsClient(...)

# Most endpoint operations require ...
token = api.authenticate('<username>', '<password>')

# And so on ...
```

That will give people a complete picture of the example and they can copy it!